addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ExtensionZest.java

```diff
@@ -573,11 +582,21 @@ public void updated(ScriptNode node) {
         ZestScriptWrapper sw = this.getZestTreeModel().getScriptWrapper(node);
         sw.incModCount();
         sw.setChanged(true);
-
         if (this.getExtScript().getScriptUI() != null
                 && this.getExtScript().getScriptUI().isScriptDisplayed(sw)) {
             // We need to do this to prevent the UI slating any changes
-            this.getExtScript().getScriptUI().displayScript(sw);
+            try {
```

Comment:
Worth doing in its own PR (if you already verified the fix).

Reply:
I can confirm that the code is returning from here: https://github.com/zaproxy/zap-extensions/pull/4768/files#diff-73e3855cfb6483aa7d55f99ceff5f2cbd5be7c4a78dd5bd1470c09ac12c6e6a7R448-R450
whenever any statement is added but it is still stealing the focus.

Reply:
Is there anything else which is response for grabbing focus?

Reply:
okay sure

Reply:
moved the focus fix into a new PR